Directory.Build.props

```diff
@@ -382,6 +381,10 @@
     <DocumentationFile Condition="'$(DocumentationFile)' == ''">$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(IsSourceProject)' == 'true' OR '$(IsReferenceAssembly)' == 'true'">
+    <Features>$(Features);nullablePublicOnly</Features>
```

Comment:
nit: do you expect to group more properties here? If not, have you considered just putting this property where it was with the condition directly on the property?

Reply:
nit: we lowercase operators

Reply:
Fixed.